e2e/create-pages.spec.ts

```diff
@@ -116,5 +116,32 @@ for (const mode of ['DEV', 'PRD'] as const) {
       ).toBeVisible();
       ({ port, stopApp } = await startApp(mode));
     });
+
+    test('api hi.txt', async () => {
+      const res = await fetch(`http://localhost:${port}/api/hi.txt`);
```

Comment:
If this is better, you might want to change define-route.spec.ts. Another PR is fine.

Reply:
yep - this is definitely better. I'll change that in another PR. 

It's better because:

- it's closer to the intended use of the api routes
- it runs WAY faster in the e2e test suite